ui/components/component-library/modal-focus/README.mdx

```diff
@@ -0,0 +1,248 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
```

Comment:
Will be removed once https://github.com/MetaMask/metamask-extension/pull/18979 is merged